impl/src/main/java/org/jboss/cdi/tck/tests/event/observer/abortProcessing/ObserverExceptionAbortsProcessingTest.java

```diff
@@ -39,6 +40,7 @@ public static WebArchive createTestArchive() {
     public static class AnEventType {
     }
 
+    @Dependent
```

Comment:
We should move bean classes (as well as the event type) out of the test class. Not needed in this PR I guess.

Reply:
I have noticed this several times throughout TCKs. While I am with you in that it is nicer to have them separate, this is a perfectly legitimate way to declare beans. Maybe having a few of these in TCKs provides us with some on-the-side testing of this fact :-)

Reply:
I actually think it's uglier to have the classes outside of the test class :-), but we need the classes outside so that testing an annotation processor based implementation is even possible. We debated that on the calls several times. But, again, not a problem of this PR!